### DISCUSSION TITLE
Broken Notification system

### DISCUSSION THREAD

**Discussion by ashmaroli** [community/community#16077] [Mobile] [answered]
Hi,
I started using the GitHub App recently on an **android device. (Android v10)**.
While I have enabled system settings to push notifications, I **never receive any notification** irrespective of concerned event (direct mention, review request, etc).

**Comment by amrehman** [comment, marked as answer]
Hi there @ashmaroli , sorry to hear that you are having some trouble.

I am going to create an issue for this for our engineering team and we will look at this asap! 

Could you please share what app version are you on?

**Reply by ashmaroli** [reply]
Hello @amrehman
Thank you for looking into this.

I **was using `v1.61.0`** at the time I started this thread. I am *currently on `v1.64.0` (updated as I'm typing this on my laptop)*.

**Comment by elisealix22** [comment]
@ashmaroli I'm sorry this is happening to you! Just to triple check, within the GitHub app if you go to: `Profile -> Settings -> Configure Notifications` are all the push types enabled within the app as well?

If so, does signing out and signing back in resolve the issue? I know that's not ideal, but trying to think what might be causing this.

**Comment by ashmaroli** [comment]
@elisealix22 I think updating from `v1.61.0` to `v1.64.0` has resolved the issue for me.
I was served a push notification when you *mentioned* me in your comment.

So @amrehman's issue ticket at the engineering div can be closed now.

Thank you very much :smile:

**Reply by elisealix22** [reply]
Ah great to hear! Thanks for the response!

**Reply by amrehman** [reply]
Awesome. Thanks @elisealix22
